luigi/rpc.py

```diff
@@ -42,32 +62,61 @@ def __init__(self, message, sub_exception=None):
         self.sub_exception = sub_exception
 
 
+class FetcherException(Exception):
+    def __init__(self, original_exc):
+        self.original_exc = original_exc
+
+
+class URLLibFetcher(object):
+    def fetch(self, full_url, body, timeout):
+        try:
+            body = urlencode(body).encode('utf-8')
+            return urlopen(full_url, body, self._connect_timeout).read().decode('utf-8')
```

Comment:
Eh this is scary. There's no such thing as a `self._connect_timeout` in this context right? Is this code path really tested?

I know I've had to disable some tests on Travis. Can you make sure a normal `tox -e py27-nonhdfs` passes before we merge this?